Makefile

```diff
@@ -48,7 +50,7 @@ testunit:
 
 .PHONY: testint
 testint:
-	pytest tests/integration
+	pytest tests/integration/${INTEGRATION_TEST_PATH}
```

Comment:
Can it still run full test suite when `INTEGRATION_TEST_PATH` isn't set?

Reply:
Yep! I checked this locally and it seems to work as expected